pkg/tsdb/cloudwatch/response_parser.go

```diff
@@ -195,7 +195,14 @@ func buildDataFrames(ctx context.Context, startTime time.Time, endTime time.Time
 
 		name := label
 		var labels data.Labels
-		if features.IsEnabled(ctx, features.FlagCloudWatchNewLabelParsing) {
+		if query.GetGetMetricDataAPIMode() == models.GMDApiModeSQLExpression {
+			metricQueryLabel := label
+			if hasStaticLabel {
+				metricQueryLabel = query.Label
+			}
+			labels = data.Labels{}
+			labels["Series"] = metricQueryLabel
```

Comment:
nit: I think you can do this as one line
```suggestion
			labels = data.Labels{"Series": metricQueryLabel}
```